appservice/src/index.ts

```diff
@@ -3,36 +3,37 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-export * from './KuduModels';
-export * from './SiteClient';
-export * from './TunnelProxy';
 export * from './confirmOverwriteSettings';
 export * from './createAppService/AppInsightsCreateStep';
 export * from './createAppService/AppInsightsListStep';
 export { AppKind, WebsiteOS } from './createAppService/AppKind';
 export * from './createAppService/AppServicePlanCreateStep';
 export * from './createAppService/AppServicePlanListStep';
+export * from './createAppService/AppServicePlanRedundancyStep';
```

Comment:
nit: Do you know what's changing all of the ordering of the exports? Do you have a different linter configuration or something?

Reply:
I assume it's from this:
![image](https://github.com/microsoft/vscode-azuretools/assets/40250218/28c4ac86-c9c2-4f3b-8151-8e8be386d6e2)

I think at some point this got turned on by default for me as I don't ever remember explicitly setting this.  It's basically just alphabetizing the imports which I feel like is good practice anyway, but I'm happy to manually change back the other imports and save without format if that's preferred

Reply:
Maybe just add that to the .vscode settings of the repo then? I don't have a strong preference, I'd just want us to align so we don't have to keep seeing diffs between export alphabetizing.

Reply:
Reverted, maybe we can discuss standardizing formatting a bit more with the team at some later date